### DISCUSSION TITLE
I want to participate in GSoC 2023, need help to get started with open source.

### DISCUSSION THREAD

**Discussion by YesShubham01** [robolectric/robolectric#7893] [General]
Hello community,
Myself Shubham, i am currently pursuing bachelor's in computer engineering at YMCA Faridabad. I have worked with java , javascript and working on android development nowadays.
I need guidance to start working on this project.
So, can you tell me more about this and guide me further about the same.

**Comment by RohanMishra315** [comment]
@YesShubham01  hey could you help me how do i copy android dependencies on local maven repo with this ./scripts/install-dependencies.rb where to find local maven repo?? pls help

Press escape to close this hovercard

**Reply by utzcoz** [reply]
@RohanMishra315 you can check https://www.baeldung.com/maven-local-repository to find common local maven paths in different OS.

**Comment by volsa** [comment]
Off-topic but why did I get a notification for this thread? I've never interacted with this project, yet received an email. Anyone else?

**Reply by AOx0** [reply]
I think I'm in the same situation. I don't remember this thread nor repository but still get this thread's notifications now and on. 

I didn't pay much attention to it tho. But now that you make the point I will proceed to unsubscribe from its notifications.

**Reply by megahomyak** [reply]
Somebody tried to scam us, and it seems like GitHub decided to subscribe us to the discussion since we were mentioned.

This image is a screenshot from my e-mail notification:
![image](https://user-images.githubusercontent.com/58137927/211465004-e962fffa-312a-4572-af64-781dd07373fd.png)

**Reply by utzcoz** [reply]
Sorry for that. The origin spam comment @ ed too many people, and everyone who were @ ed will receive notifications when there is a new comment.

**Comment by utzcoz** [comment]
@YesShubham01 I think the best approach to contribute Robolectric with code is to use it, find bugs/features, try to fix or implement it and learn core modules of Robolectric, send PRs to review and learn from discuss, and PRs are merged. If you want to contribute to shadow implementations, the largest part of Robolectric, it's better you can read AOSP source code and implement some similar logic with pure Java for Robolectric.

**Comment by utzcoz** [comment]
The Robolectric will not participate in GSoC 2023.
